.gitignore

```diff
@@ -58,4 +58,7 @@ typings/
 .env
 
 # DynamoDB Local directory
-dynamodblocal
\ No newline at end of file
+dynamodblocal
+
+# vscode
+.vscode
```

Comment:
Should this not be in your machines global .gitignore instead of the repository one ?

Reply:
Probably yes. I'll remove it